docs/tasks/includes/_node-wip.md

```diff
@@ -0,0 +1,3 @@
+:::note
+The Node.js library is being revised.  The documentation will be updated as soon as possible with the latest changes.
+:::
```

Comment:
Perhaps we should include a similar message about the JS library?

Reply:
Added!